code/modules/mob/dead/observer/observer.dm

```diff
@@ -382,9 +383,11 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 			H.remove_hud_from(usr)
 	if(!M.antagHUD)
 		to_chat(usr, "AntagHud Toggled ON")
+		create_log(MISC_LOG, "Enabled AntagHUD")
```

Comment:
Could you also log these to the `game.log` file (Either `log_misc` or `log_game`)

That was we can see if people are doing it over multiple rounds as opposed to just one.

Reply:
Ah, I had assumed `create_log()` was doing that already so I didn't add it, so sure!